core/src/main/java/lucee/runtime/functions/system/InternalRequest.java

```diff
@@ -58,8 +60,15 @@ public class InternalRequest implements Function {
 	private static final Key CONTENT_TYPE = KeyImpl.getInstance("content-type");
 	private static final Key CONTENT_LENGTH = KeyImpl.getInstance("content-length");
 
+	private static final List<String> methods = Arrays.asList(new String[] { "GET", "POST", "HEAD", "PUT", "DELETE", "OPTIONS", "TRACE", "PATCH" });
+
+
 	public static Struct call(final PageContext pc, String template, String method, Object oUrls, Object oForms, Struct cookies, Struct headers, Object body, String strCharset,
 			boolean addToken, boolean throwonerror) throws PageException {
+
+		method = method.toUpperCase().trim();
+		if (methods.indexOf(method) < 0) throw new FunctionException(pc, "_InternalRequest", 2, "method", "invalid method type [" + method + "], valid types are [" + methods.toString() + "]");
```

Comment:
That .toString() will not work as expected, you need to do the following instead ListUtil.arrayToList(methods,", ")

Reply:
@michaeloffner I've modified to use ListUtil.arrayToList() https://github.com/lucee/Lucee/pull/1799/commits/b30c0de34351b6babfbc0ad4e3e014c88a33205f

can you please review my changes?